.gitignore

```diff
@@ -6,7 +6,7 @@ build-out/
 cmake-out/
 
 # Common bazel output directories
-bazel-*
+/bazel-*
```

Comment:
If you build in one of the `quickstart/` directories this could be a problem.  There is no way to win.

Reply:
I added a rule to ignore `/google/cloud/*/quickstart/bazel-*`, which seems to cover that case too.